### PR TITLE
Full Project Pull request

### PR DESCRIPTION
**Pull request by iammouaz**
Hi there,

The Below List for what I did on this project 

- Select the API.
- Build files structure for your React app. 
- Prepare routes and navigation in your app.
- Create the tests for the application.
- Style your components to match the design provided.
- Deploy the project and test for final details.

Loom Video [Here ](https://www.loom.com/share/eccb07190f394a7c80a87d2215c7caa0)

Thank you

### CODE REVIEW

File: src/tests/components.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  it('Cryptos Components Matches', () => {
+    const cryptos = renderer
+      .create(<Provider store={store}><Cryptos /></Provider>)
+      .toJSON();
+    expect(cryptos).toMatchSnapshot();
+  });
+  test('Check the initialState of the reducer', () => {
+    expect(CryptoReducer(undefined, {})).toEqual([]);
+  });
+});
```
Comment (AshakaE) [inline src/tests/components.test.js]: The `it` block in your tests do not describe the function of the components properly.

- [x] Please write tests for all your components

File: README.md [hunk 2]
```diff
@@ -1 +1,73 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+To everyone in Microverse.
```
Comment (AshakaE) [inline README.md:69]: After following the [design guide](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)),

- [x] Please remember to acknowledge [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @iammouaz  🙋‍♀️, 

Your:
   - Project is impressive 👏 
   - Video presentation is awesome 👍 
   - Git flow ✔️ 
   - Pull request 👍 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

- [x] As per the design, please follow the [design guide](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) for the project  including:

   - Typography: font face, size, and weight
   - Layout: composition and space between elements.

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Davidosky007** [comment]
## Hi @iammouaz 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

job-welldone-gif


## Cheers and Happy coding!👏👏👏

## Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**